middleware/common/source/service/lookup.cpp

```diff
@@ -122,11 +121,18 @@ namespace casual
       {
          Trace trace{ "common::service::Lookup::operator()"};
```

Comment:
The non-blocking ditto of this already appears to wait until it gets a 'non-busy' reply, so it should work as-is.